Build warnings from typescript compilation of ember-glimmer

A fresh `ember b -prod` on master results in the following output related to the recent typescripting efforts:

```
packages/ember-glimmer/lib/components/link-to.ts(314,8): error TS6133: 'Logger' is declared but never used.
packages/ember-glimmer/lib/components/link-to.ts(315,10): error TS2300: Duplicate identifier 'assert'.
packages/ember-glimmer/lib/components/link-to.ts(315,18): error TS2300: Duplicate identifier 'deprecate'.
packages/ember-glimmer/lib/components/link-to.ts(317,10): error TS2300: Duplicate identifier 'assert'.
packages/ember-glimmer/lib/components/link-to.ts(317,18): error TS2300: Duplicate identifier 'deprecate'.
packages/ember-glimmer/lib/components/link-to.ts(658,7): error TS2554: Expected 2-3 arguments, but got 1.
```

--- a/packages/ember-glimmer/lib/components/link-to.ts
+++ b/packages/ember-glimmer/lib/components/link-to.ts
@@ -311,8 +311,6 @@
   @public
 */
 
-import Logger from 'ember-console';
-import { assert, deprecate } from 'ember-debug';
 import { DEBUG } from 'ember-env-flags';
 import { assert, deprecate, warn } from 'ember-debug';
 import {
@@ -655,7 +653,7 @@ const LinkComponent = EmberComponent.extend({
 
     if (get(this, 'loading')) {
       // tslint:disable-next-line:max-line-length
-      warn('This link-to is in an inactive loading state because at least one of its parameters presently has a null/undefined value, or the provided route name is invalid.');
+      warn('This link-to is in an inactive loading state because at least one of its parameters presently has a null/undefined value, or the provided route name is invalid.', false);
       return false;
     }
 

